test/ibmq/test_ibmq_job.py

```diff
@@ -400,16 +401,19 @@ def test_retrieve_jobs_start_datetime(self, provider):
         """Test retrieving jobs created after a specified datetime."""
         backend = provider.get_backend('ibmq_qasm_simulator')
         past_month = datetime.now() - timedelta(days=30)
-        past_month_str = past_month.strftime('%Y-%m-%dT%H:%M:%S.%fZ')
+        past_month_str = datetime_to_str(past_month)
 
         job_list = provider.backends.jobs(backend_name=backend.name(),
                                           limit=5, skip=0, start_datetime=past_month)
         self.assertTrue(job_list)
-        for i, job in enumerate(job_list):
-            self.assertTrue(job.creation_date() >= past_month_str,
-                            '{}) job creation_date {} is not '
-                            'greater than or equal to past month: {}'
-                            .format(i, job.creation_date(), past_month_str))
+        for job in job_list:
+            job_id = job.job_id()
+            with self.subTest(job_id=job_id):
```

Comment:
Do you really need subTest here? You're just verifying the results, not testing with new parameters.